node/wipe_appwrite_bucket/README.md

```diff
@@ -0,0 +1,46 @@
+# 🚮 Wipe Appwrite Bucket
+
+This function deletes every file from a specified Appwrite Storage Bucket.
+
+## 🤖 Documentation
+
+**Before:**
+
+![Appwrite Storage Bucket containing 4 different files](assets/before.png)
+
+**After:**
+
+![The same Appwrite Storage Bucket which is now empty](assets/after.png)
```

Comment:
To be consistent with the other examples, please leave the images out of version control. Instead, the screenshot would be good in the PR.